### PR TITLE
Update alert pages

### PR DESCRIPTION
**Pull request by zapbot**
Signed-off-by: zapbot <12745184+zapbot@users.noreply.github.com>

### CODE REVIEW

File: site/content/docs/alerts/43.md [hunk 1]
```diff
@@ -7,27 +7,7 @@ alertcount: 1
...
-
-When the set of acceptable objects, such as filenames or URLs, is limited or known, create a mapping from a set of fixed input values (such as numeric IDs) to the actual filenames or URLs, and reject all other inputs.
-
-Run your code in a 'jail' or similar sandbox environment that enforces strict boundaries between the process and the operating system. This may effectively restrict which files can be accessed in a particular directory or which commands can be executed by your software.
-
-OS-level examples include the Unix chroot jail, AppArmor, and SELinux. In general, managed code may provide some protection. For example, java.io.FilePermission in the Java SecurityManager allows you to specify restrictions on file operations.
-
-This may not be a feasible solution, and it only limits the impact to the operating system; the rest of your application may still be subject to compromise."
+solution: "Ensure that arbitrary files specified by the user are not included in the output"
```
Comment (kingthorin) [inline site/content/docs/alerts/43.md:10]: Should have a period?

File: site/content/docs/alerts/43.md [hunk 2]
```diff
@@ -39,10 +19,4 @@ alerttags:
...
-
-Even if the web server properly restricts Path Traversal attempts in the URL path, a web application itself may still be vulnerable due to improper handling of user-supplied input. This is a common problem of web applications that use template mechanisms or load static text from files. In variations of the attack, the original URL parameter value is substituted with the file name of one of the web application's dynamic scripts. Consequently, the results can reveal source code because the file is interpreted as text instead of an executable script. These techniques often employ additional special characters such as the dot (".") to reveal the listing of the current working directory, or "%00" NULL characters in order to bypass rudimentary file extension checks.
+The source code for the current page was disclosed by the web server
```
Comment (kingthorin) [inline site/content/docs/alerts/43.md:22]: Should have a period?

**Review by kingthorin — COMMENTED** [review]
I guess this is based on messages.properties content, but they could use tweaking I guess?

File: site/content/docs/alerts/43.md (hunk 1 above)
Comment (thc202) [reply site/content/docs/alerts/43.md:10]: They can be added, though the loss of the previous description/solution is more worrying. Not much to do right now anyway.

**Review by thc202 — APPROVED** [review]

File: site/content/docs/alerts/43.md (hunk 1 above)
Comment (kingthorin) [reply site/content/docs/alerts/43.md:10]: I seem to recall a PR that removed some overly verbose elements. I'll check this later today.

Comment (thc202) [reply site/content/docs/alerts/43.md:10]: This scan rule is using different sources for the description/solution of the scan rule and the alerts raised.

Comment (kingthorin) [reply site/content/docs/alerts/43.md:10]: Oh okay. Thanks, I hadn't been able to check yet.

**Comment by thc202** [comment]
Closing, we will have to change the details one way or another.

**Pull request closed without merging**